tasks/src/robocorp/tasks/cli.py

```diff
@@ -9,35 +9,54 @@
 code which must be automatically logged is not imported prior the the `cli.main`
 call.
 """
+
 import os
 import sys
 import warnings
+from importlib.metadata import PackageNotFoundError, version as get_version
 from typing import List, Optional, Protocol
 
+from packaging import version as version_parser
+
 
-def inject_truststore():
-    # Use certificates from native storage (if `truststore` installed)
+def _inject_truststore():
+    # Use certificates from the native storage. (if `truststore` installed and minimum
+    #  interpreter/pip versions are met)
 
-    # _RC_TEST_USE_TRUSTSTORE env var is only used to avoid unwanted warnings in tests
-    # and should not be used for other purposes
+    # NOTE(OvidiuCode): `_RC_TEST_USE_TRUSTSTORE` env var is only used to avoid
+    #  unwanted warnings under tests, and should not be used for other purposes.
```

Comment:
In PyCharm the contiguous comment blocks prefer some leading space. Replace `NOTE` with `TODO` / `FIXME` to observe the highlight I'm talking about, which won't happen on the following rows without the extra initial space.